doc/styles/Button.md

```diff
@@ -19,167 +19,117 @@ IsDefaultStyle\*: Styles in this column will be set as the default implicit styl
-| `TextButtonStateLayerBackgroundFocused`               | `SolidColorBrush` | `PrimaryFocusedBrush`              |
-| `TextButtonStateLayerBackgroundPointerFocused`        | `SolidColorBrush` | `PrimaryFocusedBrush`              |
-| `IconButtonForeground`                                | `SolidColorBrush` | `OnSurfaceVariantBrush`            |
-| `IconButtonForegroundDisabled`                        | `SolidColorBrush` | `OnSurfaceLowBrush`                |
-| `IconButtonEllipseFillPointerOver`                    | `SolidColorBrush` | `PrimaryHoverBrush`                |
-| `IconButtonEllipseFillPressed`                        | `SolidColorBrush` | `PrimaryPressedBrush`              |
-| `IconButtonEllipseFillFocused`                        | `SolidColorBrush` | `PrimaryFocusedBrush`              |
-| `ButtonMinHeight`                                     | `Double`          | 40                                 |
-| `ButtonMinWidth`                                      | `Double`          | 40                                 |
-| `ButtonIconMinWidth`                                  | `Double`          | 18                                 |
-| `ElevatedButtonElevation`                             | `Double`          | 1                                  |
-| `ElevatedButtonElevationDisabled`                     | `Double`          | 0                                  |
-| `ButtonElevation`                                     | `Double`          | 0                                  |
-| `IconButtonOpacityVisibleState`                       | `Double`          | 1                                  |
-| `IconButtonOpacityHiddenState`                        | `Double`          | 0                                  |
-| `ButtonPadding`                                       | `Thickness`       | 16,0                               |
-| `TextButtonPadding`                                   | `Thickness`       | 12,0                               |
-| `TextButtonIconMargin`                                | `Thickness`       | 0,0,8,0                            |
-| `ButtonBorderThickness`                               | `Thickness`       | 0                                  |
-| `OutlinedButtonBorderThickness`                       | `Thickness`       | 1                                  |
-| `ElevatedButtonMargin`                                | `Thickness`       | 0,0,0,1                            |
-| `ElevatedButtonDisabledMargin`                        | `Thickness`       | 0                                  |
-| `ButtonContentMargin`                                 | `Thickness`       | 8,0                                |
-| `ButtonMargin`                                        | `Thickness`       | 0                                  |
-| `ButtonCornerRadius`                                  | `CornerRadius`    | 20                                 |
```

Comment:
@Xiaoy312, @kazo0 We need to update https://github.com/unoplatform/uno.themes/blob/master/doc/material-migration.md to mention the breaking changes